### ISSUE TITLE
[FEATURE] Feature command: invalid feature name error message variation

### ISSUE THREAD

**Issue by C0ZEN** [Sonia-corporation/sonia-discord#821]
**Describe the solution you'd like**
It could be in this specific case a good place to create variables message based on the number of errors the user entered.
This must be generic enough to remember in this case the last user input.

**Like:**

- First error is the default response
- Second time the same error is another response
- Etc

**To do a bit of conception:**

The user input should be remembered.
Firebase should be used then.
I think this is better to remember the user per guild and not on Discord entirely.
Since in the future Sonia will has more and more features configured for the given guild this must be done only inside it.
So basically, store a map of users per guild (/guilds).
Inside the user we should have a last command message.
Also I think this is a good starting point for another feature required before this one:
- Had an enum for each "path" inside the commands taken
- This will help to identify the case easily and also will help in the future with the stats
